server/src/main/java/org/apache/iotdb/db/service/TSServiceImpl.java

```diff
@@ -603,9 +602,15 @@ public TSExecuteStatementResp executeStatement(TSExecuteStatementReq req) {
               physicalPlan,
               req.fetchSize,
               req.timeout,
-              sessionManager.getUsername(req.getSessionId()),
+              req.getSessionId(),
               req.isEnableRedirectQuery())
-          : executeUpdateStatement(physicalPlan, req.getSessionId());
+          : executeUpdateStatement(
+              statement,
+              req.statementId,
+              physicalPlan,
+              req.fetchSize,
+              req.timeout,
+              req.getSessionId());
```

Comment:
use non query

Reply:
I think the name `executeUpdateStatement` is ok.

First, `executeUpdate` is an interface of JDBC, we can keep this name at the entry of the method.

Second, `executeUpdate` may have two tasks to execute, one is` executeSelectIntoStatement`, the other is `executeNonQueryStatement`. Two methods are very different.


![image](https://user-images.githubusercontent.com/30497621/127081040-e08d0ffb-32c8-40ff-9d97-2e5a4bdb5a4e.png)